episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+`File > Open Sample > napari builtins > Skin (RGB)`
+
+![](fig/skin-napari.png){alt="A screenshot of an H+E slide of skin layers 
+in Napari"}
+
+This image is a hematoxylin and eosin stained slide of dermis and epidermis 
+(skin layers). Let's check its shape:
+
+```python
+image = viewer.layers["skin"].data
+print(image.shape)
+```
+
+```output
+# (y, x, c)
+(960, 1280, 3)
+```
+
+In this case, the channels aren't separated out into different image layers, 
+but rather shown combined together in a single image. If you hover your mouse 
+over the image, you should see three pixel values printed in the bottom left 
+representing (R, G, B).
+
+![](fig/rgb-values.png){alt="A screenshot of an H+E slide of skin layers in 
+Napari, highlighting the (R,G,B) values"}
+
+We can see these different channels more clearly if we right click on the 'skin' 
+image layer and select:  
+`Split RGB`
+
+This shows the red, green and blue channels as separate image layers. Try 
+inspecting each one individually by clicking the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility.svg
+){alt="A screenshot of Napari's eye button" height='30px'} icons to hide the 
+other layers.
+
+We can understand these RGB pixel values better by opening a different sample 
+image. Remove all layers, then select:  
+`File > Open Sample > napari builtins > Colorwheel (RGB)`
+
+![](fig/colorwheel-napari.png){alt="A screenshot of a colorwheel in Napari"}
+
+This image shows an RGB colourwheel - try hovering your mouse over different 
+areas, making note of the (R, G, B) values shown in the bottom left of Napari. 
+You should see that moving into the red area gives high values for R, and low 
+values for B and G. Equally, the green area shows high values for G, and the 
+blue area shows high values for B. The red, green and blue values are mixed 
+together to give the final colour.
+
+
+RGB images are very common in day-to-day life - for example, photos taken with a 
+camera or phone will be RGB. This being said, they are less common in microscopy 
+- although there are certain research fields and types of microscope that 
+commonly use RGB. A key example is imaging of tissue sections for histology or 
+pathology. You will also often use RGB images when preparing figures for papers 
+and presentations - RGB images can be opened in all imaging software (not just 
+scientific research focused ones), so are useful when preparing images for 
+display. As usual, always make sure you keep a copy of your original raw data! 
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+## Reading RGB in Napari
+
+How does Napari know an image is RGB? If an image's final dimension has a length 
+of 3 or 4, Napari will assume it is RGB and [display it as such
+](https://napari.org/stable/howtos/layers/image.html#viewing-rgb-vs-luminance-grayscale-images). 
+If loading an image from the console, you can also manually set it to load 
+as rgb:
```

Comment:
This doesn't work quite as I would expect it to based on the text here. With rgb not set or True I get a single channel colour image of an astronaut. If I set rgb=False I was expecting 3 layers, but I only got a single layer greyscale image.